scripts/travis-coverage.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import tempfile, subprocess, sys, os, ntpath, re
+from os.path import exists, isdir, isfile
+
+_ERR_PREFIX = '\033[31mtravis-coverage.py: error: '
+_WARN_PREFIX = '\033[31mtravis-coverage.py: warning: '
+_BOLD_PREFIX = '\033[1m'
+_BLUE_PREFIX = '\033[34m'
+
+f = str(sys.argv[1])
+if not (exists(f) and isfile(f)):
+    sys.exit(_ERR_PREFIX + f + ' does not exist!\033[0m')
+
+_DIR = tempfile.mkdtemp()
+_COMMANDS = 'echo "Test(\\"' + f + '\\");;\n'
+_COMMANDS += '''UncoverageLineByLine();;
+LoadPackage(\\"profiling\\", false);;
+filesdir := \\"''' + os.getcwd() + '''/gap/\\";;\n'''
+_COMMANDS += 'outdir := \\"' + _DIR + '\\";;\n'
+_COMMANDS += 'x := ReadLineByLineProfile(\\"' + _DIR + '/profile.gz\\");;\n'
+_COMMANDS += 'OutputAnnotatedCodeCoverageFiles(x, filesdir, outdir);"'
+
+pro1 = subprocess.Popen(_COMMANDS, stdout=subprocess.PIPE, shell=True)
+_RUN_GAP = '../../bin/gap.sh -A -q -r -m 1g -T --cover ' + _DIR + '/profile.gz'
+
+try:
+    pro2 = subprocess.Popen(_RUN_GAP,
+                            stdin=pro1.stdout,
+                            shell=True)
+    pro2.wait()
+except KeyboardInterrupt:
```

Comment:
Not completely sure that this line is required for a script running on travis, but it won't do any harm either 👍